lib/service_check.pm

```diff
@@ -11,13 +11,19 @@
 # Maintainer: GAO WEI <wegao@suse.com>
 
 package service_check;
-use base Exporter;
-use Exporter;
+
+use Exporter 'import';
 use testapi;
 use utils;
 use base 'opensusebasetest';
 use strict;
 use warnings;
+use File::Basename;
+BEGIN {
+    unshift @INC, dirname(__FILE__) . '/../tests/console';
```

Comment:
I don't like the idea of including tests inside a lib/* file.

Reply:
we can move lib/service_check.pm  to the test/console to avoid this.

Reply:
I'm a bit lost which one you should move or not...
Maybe this could help:
1. if there are multiple users of the functions in service_check.pm, keep it in `lib/`
2. move functions from http_srv.pm to a new lib file, which are (or might be) used by more then one test-script
3. Do not place perl modules in `lib/`, which gets loaded with `loadtest`